src/debug/inc/arm/primitives.h

```diff
@@ -148,7 +148,7 @@ inline PRD_TYPE CORDbgGetInstruction(UNALIGNED CORDB_ADDRESS_TYPE* address)
 {
     LIMITED_METHOD_CONTRACT;
 
-    ULONG ptraddr = dac_cast<ULONG>(address);
+    size_t ptraddr = dac_cast<size_t>(address);
```

Comment:
`CORDB_ADDRESS` ?

Reply:
`CORDB_ADDRESS` is defined as `ULONG64` which does not work under `DACCESS_COMPILE`
```
/opt/code/src/inc/daccess.h:2020:13: error: static_assert failed "is_base_of constraint violation"
            static_assert((std::is_base_of<__TPtrBase, T>::value), "is_base_of constraint violation");
            ^             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/code/src/inc/daccess.h:2294:39: note: in instantiation of member function 'dac_imp::makeDacInst<unsigned long long>::fromTaddr' requested here
    return dac_imp::makeDacInst<Tgt>::fromTaddr(addr);
                                      ^
/opt/code/src/debug/inc/arm/primitives.h:151:29: note: in instantiation of function template specialization 'dac_cast<unsigned long long, const unsigned char *>' requested here
    CORDB_ADDRESS ptraddr = dac_cast<CORDB_ADDRESS>(address);
                            ^
1 error generated.
```

Reply:
This may just need to be a regular cast. I do not think this needs to be `dac_cast`.